FiducialRegistrationWizard/Logic/vtkPointDistanceMatrix.h

```diff
@@ -0,0 +1,49 @@
+#ifndef __vtkPointDistanceMatrix_h
+#define __vtkPointDistanceMatrix_h
+
+#include <vtkDoubleArray.h>
+#include <vtkObject.h>
+#include <vtkPoints.h>
+
+// At its most basic level, this is a Wrapper class for storing a matrix of
```

Comment:
It would be nicer to implement this class as proper VTK filter (takes two vtkPoints objects as input, provides a vtkDoubleArray as output) - if you are interested in learning how to implement such filters.